modules/system/overrides.js

```diff
@@ -147,7 +147,7 @@ export default function () {
 
 
   Token.prototype.incrementCondition = async function (effect, { active, overlay = false } = {}) {
-    const existing = this.actor.actorEffects.find(e => e.conditionKey === effect.id);
+    const existing = this.actor.effects.contents.find(e => e.conditionKey === effect.id);
```

Comment:
this could probably be changed to `this.actor.hasCondition(effect.id)`

Reply:
changed